### PR TITLE
Fixes #5 

### LINKED ISSUE #5: Doesn't work on Windows
@krzkaczor, Hyper 1.0.0 was released with Windows support.

This attractive plugin doesn't seem to work. It doesn't throw any errors to the console.

Let's try and work it out together?

### PR DESCRIPTION
**Pull request by bboysathish**
Adds Support for Windows + Linux

### CODE REVIEW

**Comment by krzkaczor** [comment]
First of all thanks for PR! I have few suggestions. It's a little bit tangled now and I think it won't work on Mac anymore. What would I suggest:
1. create function `isPasting` which takes keyboard event as a argument
2. `isPasting` checks for platform (mac / rest of the world (win/linux))
3. `if mac then return e.metaKey && e.keyCode === 86`
4. `if not mac then return e.ctrlKey && e.keyCode === 86`

What do you think about this solution?

**Comment by bboysathish** [comment]
Is it? I thought handling this way would make it more generic and will work on all systems, I am a windows user, I don't have mac to test. if you have already tested it and it is not working, I can work on your solution and push @krzkaczor

**Comment by krzkaczor** [comment]
Your solution would allow pasting with ctrl on macs. We don't want to do that ;) I think that explicit if statement for platform check will be the most readable way of doing it.

**Comment by bboysathish** [comment]
Okie will do the change and push @krzkaczor .

**Comment by bboysathish** [comment]
ping @krzkaczor changed!

File: src/index.js [hunk 1]
```diff
@@ -1,6 +1,16 @@
 const { clipboard } = require('electron')
 const sanitizeInput = require('./sanitizeShellInput')
 
+const isPasting = e => {
+  const isDarwin = process.platform === "darwin";
+  const key = e.which || e.keyCode; // keyCode detection
```
Comment (krzkaczor) [inline src/index.js]: do we need e.which? according to [this](http://stackoverflow.com/questions/4471582/javascript-keycode-vs-which) it's only for *some* browsers. In electron i don't think that we need to care about it.

File: src/index.js [hunk 2]
```diff
@@ -1,6 +1,16 @@
...
+  const ctrl = e.ctrlKey ? e.ctrlKey : ((key === 17) ? true : false); // ctrl detection
```
Comment (krzkaczor) [inline src/index.js]: do we need that additional check for `key === 17`? Isn't `e.ctrlKey` enough?

File: src/index.js (hunk 1 above)
Comment (bboysathish) [inline src/index.js]: Fixed.

File: src/index.js (hunk 2 above)
Comment (bboysathish) [inline src/index.js]: Over protection :) But on second thought removed it.

File: src/index.js [hunk 3]
```diff
@@ -3,8 +3,8 @@ const sanitizeInput = require('./sanitizeShellInput')
 
 const isPasting = e => {
   const isDarwin = process.platform === "darwin";
-  const key = e.which || e.keyCode; // keyCode detection
-  const ctrl = e.ctrlKey ? e.ctrlKey : ((key === 17) ? true : false); // ctrl detection
+  const key = e.keyCode; // keyCode detection
+  const ctrl = e.ctrlKey ? e.ctrlKey : false; // ctrl detection
```
Comment (krzkaczor) [inline src/index.js]: this is basically no op ;) just use `e.ctrlKey` directly on line 11

**Comment by krzkaczor** [comment]
Thanks for your hard work! I am gonna release new version in few moments.

**Comment by bboysathish** [comment]
@krzkaczor Thank you 👍

**Comment by krzkaczor** [comment]
@bboysathish one more thing. After your changes linting failed - we don't have CI setup so I couldn't spot it. I fixed it on master. In future, you can run `npm test` locally to run all tests. Btw. it's released.

**Pull request merged**